src/main/java/com/smartbear/postman/PostmanImporter.java

```diff
@@ -98,11 +98,12 @@ public PostmanImporter(TestCreator testCreator) {
 
     public WsdlProject importPostmanCollection(WorkspaceImpl workspace, String filePath) {
         WsdlProject project = null;
-        File jsonFile = new File(filePath);
         String postmanJson = null;
+        UrlClientLoader loader = new UrlClientLoader(filePath);
```

Comment:
Probably we should use at least ``setCursor(Cursor.getPredefinedCursor(Cursor.WAIT_CURSOR))`` here.

Reply:
@Alexander-Sviridov I think it should be inside the UrlClientLoader class then, isn't is?

Reply:
No, cause potentially it can be used in some background tasks or when a test is executed. For example, see ``XmlRpc.Call`` in readyapi-free-os.